checkout/session/client.go

```diff
@@ -127,7 +129,22 @@ func (c Client) ListLineItems(id string, listParams *stripe.CheckoutSessionListL
 	}
 }
 
-type LineItemIter = lineitem.Iter
+// LineItemIter is an iterator for line items.
+type LineItemIter struct {
```

Comment:
Still unclear why this moved here.

Reply:
Generating iterators as part of clients that return them is the standard behavior. We had a special case for this one iterator. 

Examples of others:

https://github.com/stripe/stripe-go/blob/13752c6a5e72d07d4ae5d1894763bb36c62dedf9/creditnote/client.go#L186
https://github.com/stripe/stripe-go/blob/b8b156bcee1559adff7d4249f702bfe110077c41/paymentlink/client.go